### DISCUSSION TITLE
Getting error While deploying Certificate manager using CDK

### DISCUSSION THREAD

**Discussion by GirishMadheswaran** [aws/aws-cdk#22669] [Q&A]
Hi, I am getting the below error while trying to create the Certificate manager in CDK, getting error in domainName,
I have used "hello.example.com"  as a domain name.Anyone please help me regarding this or guide me how to use the domainName and done a certificate Validation,I have used the route53 Zone name as a  "example.com" as mentioned in the CDK docs- https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_certificatemanager.CertificateProps.html

20:41:31 | CREATE_FAILED        | AWS::CertificateManager::Certificate | certificatemanager4BF1F6CE
1 validation error detected: Value 'certificatemanager' at 'domainName' failed to satisfy constraint: Member must satisfy regular expression pattern: ^(\*\.)?(((?!-)[A-Za-z0-9-]{0,62}[A-Za-z0-9])\.)+((?!-)[A-Za-z0-9-]{1,62}[A-Za-z0-9])$ (Service: AWSCertificateManager; Status Code: 400; Error Code: ValidationException; Request ID: 1b5f456e-0413-4b24-a5de-3bec1233bd9d; Proxy: null)

**Comment by pahud** [comment]
It seems the value of `domainName` is `certificationmanager`, which is invalid. I believe the value should be something like `hello.example.com`.

**Reply by GirishMadheswaran** [reply]
I also tried with that `hello.example.com` again i am getting the same error `domainName` is failed to satisfy constraint, may be there is anything relation to the `hostedzone` domain which we are giving for the certificate validation
